src/utils/config.js

```diff
@@ -1,5 +1,12 @@
 import { getQueryParam } from './getQueryParam';
 
+const oidcConfigShared = {
+  clientId: process.env.REACT_APP_OIDC_CLIENT_ID,
```

Comment:
how does `REACT_APP_OIDC_CLIENT_ID` get set and passed in?

Reply:
hmmm good point. I need to add it to the build kite pipeline, as well as the login for npm to be able to get the @virtru/oidc client

Reply:
I've added it to the pipeline.